lang/cs/Org.Apache.REEF.Client/Org.Apache.REEF.Client.csproj

```diff
@@ -72,12 +88,23 @@ under the License.
       <Project>{75503f90-7b82-4762-9997-94b5c68f15db}</Project>
       <Name>Org.Apache.REEF.Examples</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Org.Apache.REEF.Bridge.JAR\Org.Apache.REEF.Bridge.JAR.csproj">
```

Comment:
This is actually a leftover from a test. I removed the dependency completely, as the client doesn't need the JARs. `HelloREEF` needs them, though. And I added the reference there using the above code.